docs/DOCKER.rst

```diff
@@ -96,11 +88,22 @@ new.host.com, for example, specify the endpoint with:
 
 .. code-block:: shell
 
-    $ docker run -d --name cloudserver -p 8000:8000 -e ENDPOINT=new.host.com scality/cloudserver
+    $ docker run -d --name cloudserver -p 8000:8000 -e ENDPOINT=new.host.com zenko/cloudserver
 
 .. note:: On Unix-like systems (Linux, OS X, etc.) edit /etc/hosts
    to associate 127.0.0.1 with new.host.com.
 
+REMOTE_MANAGEMENT_DISABLE=1
```

Comment:
keeping in line with the other env variable title formats, we could probably just title this section as `REMOTE_MANAGEMENT_DISABLE`

Reply:
totally